main.config.js

```diff
@@ -201,120 +201,146 @@ export const config = {
+      //     number: '03',
+      //     type: 'number',
+      //   },
+      //   {
+      //     name: 'Black Book',
+      //     description: 'Display the statistics of company employees',
+      //     href: '/project-black-book',
+      //     number: '04',
+      //     type: 'number',
+      //   },
+      //   {
+      //     name: 'TheCheckNetwork',
+      //     description: 'SEO-optimization as a service',
+      //     href: '/project-the-check-network',
+      //     number: '05',
+      //     type: 'number',
+      //   },
+      //   {
+      //     name: 'Wargaming',
+      //     description: 'Economic wars game',
+      //     href: '/project-wargaming',
+      //     number: '06',
+      //     type: 'number',
+      //   },
+      //   {
+      //     name: 'Knote',
+      //     description: 'A tool for online collaboration and task-management',
+      //     href: '/project-knote',
+      //     number: '07',
+      //     type: 'number',
+      //   },
+      //   {
+      //     name: 'My Grass Growth',
+      //     description: 'Keep you cows healthy and well-fed',
+      //     href: '/project-my-grass-growth',
+      //     number: '08',
+      //     type: 'number',
+      //   },
+      //   {
+      //     name: 'WorkBooking',
+      //     description: 'Get hired or find someone to do the job for you',
+      //     href: '/project-workbooking',
+      //     number: '09',
+      //     type: 'number',
+      //   },
+      //   {
+      //     name: 'BarFrogs',
+      //     description: 'Find an event you are interested in',
+      //     href: '/project-barfrogs',
+      //     number: '10',
+      //     type: 'number',
+      //   },
+      //   {
+      //     name: 'Prix',
+      //     description: 'Increasing loyalty of existing clients',
+      //     href: '/project-prix',
+      //     number: '11',
+      //     type: 'number',
+      //   },
+      //   {
+      //     name: 'Tetrami',
+      //     description: 'Online discounting',
+      //     href: '/project-tetrami',
+      //     number: '13',
+      //     type: 'number',
+      //   },
+      //   {
+      //     name: 'Phenix',
+      //     description: 'Online video-streaming',
+      //     href: '/project-phenix',
+      //     number: '14',
+      //     type: 'number',
+      //   },
+      //   {
+      //     name: 'Plutio',
+      //     description: 'Business management platform',
+      //     href: '/project-plutio',
+      //     number: '15',
+      //     type: 'number',
+      //   },
+      //   {
+      //     name: 'SelfLeaders',
+      //     description: 'Personal Growth Tool',
+      //     href: '/project-self-leaders',
+      //     number: '16',
+      //     type: 'number',
+      //   },
+      //   {
+      //     name: 'Pree',
+      //     description: 'Stop losing your stuff',
+      //     href: '/project-pree',
+      //     number: '17',
+      //     type: 'number',
+      //   },
+      // ],
```

Comment:
@dmytropanontko , do we need to keep this commented out code?

Reply:
@dmytropanontko This code was used to display works in the navigation. I think he's no longer needed there.